test/fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2023 Intel Corporation
+# Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+# See LICENSE.TXT
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+function(add_fuzz_test name)
+    set(TEST_TARGET_NAME fuzztest-${name})
+    add_executable(${TEST_TARGET_NAME}
+        ${ARGN})
+    target_link_libraries(${TEST_TARGET_NAME}
+        PRIVATE
+        ${PROJECT_NAME}::loader
+        ${PROJECT_NAME}::headers
+        ${PROJECT_NAME}::common
+        -fsanitize=fuzzer)
+    add_test(NAME ${TEST_TARGET_NAME}
+        COMMAND ${TEST_TARGET_NAME} -max_total_time=600 -seed=1 -shrink=1 -verbosity=1
+        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+    set_tests_properties(${TEST_TARGET_NAME} PROPERTIES
+        LABELS "fuzz"
+        ENVIRONMENT
+            "XPTI_TRACE_ENABLE=1"
+            "XPTI_FRAMEWORK_DISPATCHER=$<TARGET_FILE:xptifw>"
+            "XPTI_SUBSCRIBERS=$<TARGET_FILE:collector>"
+            "UR_ADAPTERS_FORCE_LOAD=\"$<TARGET_FILE:ur_adapter_null>\""
+            "UR_ENABLE_LAYERS=UR_LAYER_TRACING")
+    target_compile_options(${TEST_TARGET_NAME} PRIVATE -g -fsanitize=fuzzer)
```

Comment:
You use `-fsanitize=fuzzer` here and in `target_link_libraries`, is that intentional?